Controller/Payment/Cancel.php

```diff
@@ -47,8 +55,11 @@ public function execute()
         try {
             $order = $this->orderService->getOrderByReference();
             $this->orderService->cancelTwoOrder($order);
-            $message = 'Your invoice purchase with Two has been cancelled. The cart will be restored.';
-            throw new LocalizedException(__($message));
+            $message = __(
+                'Your invoice purchase with %1 has been cancelled. The cart will be restored.',
```

Comment:
Assuming the %1 is being dynamically replaced here.

Reply:
Yep the second param `$this->configRepository->getProvider()` fetches the provider name from config. This is defined in `etc/config.xml`.